### PR TITLE
Don't map glyphs to Unicode combined Thai character (bug1046314)

### PR DESCRIPTION
**Pull request by Snuffleupagus**
This is a very tentative patch, that fixes https://bugzilla.mozilla.org/show_bug.cgi?id=1046314.
Even though the Thai Unicode character `0x0E33` has a length of `1`, when copied I noticed that it's made up of "two" characters. I'm assuming that this is what's causing the problem in PDF file in the bug, hence I'm thinking that we can simply ignore that spot when creating the charCode map.

**Edit:** Fixes one of the bugs in #5647.

### CODE REVIEW

File: test/test_manifest.json [hunk 1]
```diff
@@ -529,6 +529,13 @@
        "rounds": 1,
        "type": "load"
     },
+    {  "id": "bug1046314",
+       "file": "pdfs/bug1046314.pdf",
+       "md5": "fc658439f44cd2dd27c8bee7e7a8344e",
+       "rounds": 1,
+       "link": false,
+       "type": "eg"
```
Comment (timvandermeij) [inline test/test_manifest.json]: This should be type `eq`.

Comment (Snuffleupagus) [inline test/test_manifest.json]: Whoops; fixed now :-)

**Comment by CodingFabian** [comment]
this does not feel right to me. 0x0E33 is one single character.could it be that the pdf-creator substituted this char?

**Pull request closed without merging**